src/traces/scattergl/attributes.js

```diff
@@ -37,9 +37,13 @@ var attrs = module.exports = overrideAll({
             'this trace\'s (x,y) coordinates.'
         ].join(' ')
     }),
+
+    textposition: scatterAttrs.textposition,
+    textfont: scatterAttrs.textfont,
+
     mode: {
         valType: 'flaglist',
-        flags: ['lines', 'markers'],
+        flags: ['lines', 'markers', 'text'],
```

Comment:
Now that scattergl supports on-graph, we can add `hovertext` like in svg scatter traces:

https://github.com/plotly/plotly.js/blob/6ac47eca310d15e412cf9f6aa41652e0e33501e7/src/traces/scatter/attributes.js#L91-L104

Reply:
done in https://github.com/plotly/plotly.js/commit/959a5554794586e941576e553de6b7e3cbfb74d3